postman_collection.json

```diff
@@ -88,6 +88,84 @@
 					},
 					"response": []
 				},
+				{
+					"name": "/user/login tagbot",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"function setEnvForTest(token) {",
+									"    var moment = require('moment');",
+									"    // Saving token for future requests",
+									"    pm.environment.set(\"tagbotToken\", token);",
+									"    // Setting dates for future requests",
+									"    var shortBeginDate = moment().subtract(1, 'month');",
```

Comment:
I don't think you need to duplicate date related code as it is already executed and set in env in trackit login.